.gitignore

```diff
@@ -1,25 +1,26 @@
 .DS_Store
-dist
-builds/*
-tasks/builds
-coverage
 node_modules
-npm-debug.log
-npm-debug.log.*
-/package-lock.json
-thumbs.db
-!.gitkeep
-.vscode/launch.json
-yarn-error.log
```

Comment:
are you sure these can all be deleted?

Reply:
Some of them are still in. Let's add again when we see fit to avoid polution